sphinx/source/docs/user_guide/concepts.rst

```diff
@@ -135,6 +135,49 @@ Another way to generate various kinds of output is by using Bokeh's powerful
 
 .. _userguide_interfaces:
 
+Bokeh settings
+--------------
+
+There are various global settings that influence how Bokeh operates.
+
+There are several methods you can use to change Bokeh's configuration. Directly
+in the Python code, in a YAML configuration file, or with environment variables,
+for example. A full list of all available settings and how to change them is
+available at :ref:`bokeh.settings`.
+
+Some of most useful settings are:
+
+* ``browser`` (environment variable ``BOKEH_BROWSER``): Set this configuration
+  value to the browser you want Bokeh to use (for example when calling
+  :func:`~bokeh.io.show`). Valid values are any of the predefined browser names
+  of the Python webbrowser_ module. For example: ``chromium-browser`` or
+  ``windows-default``. You can also set this variable to the full path of your
+  browser. For example:
+
+  .. code-block:: PowerShell
+
+      $env:BOKEH_BROWSER="C:/Program\ Files/Google/Chrome/Application/chrome.exe %s &"
+
+* ``legacy`` (environment variable ``BOKEH_LEGACY``): Set this configuration
+  value to ``True`` to enable support for legacy browsers (IE and phantomjs).
```

Comment:
@tcmetzger it might be worth seeing how these render as definition lists? e.g. 
```
``legacy`` (environment variable ``BOKEH_LEGACY``)
    Set this configuration value to ``True`` to enable support for legacy 
    browsers (IE and phantomjs).
```
I just generally find big bulleted blocks very cumbersome / ugly / harder to read

Reply:
Very good idea!